src/components/footer/footer.jsx

```diff
@@ -1,8 +1,77 @@
 import styles from "@/styles/components/footer/footer.module.scss";
 
+import {useTranslations} from 'next-intl';
+import Image from "next/image";
+import Link from "next/link";
+
+import React, { Component } from "react";
+import { AiOutlineInstagram } from "react-icons/ai";
+import { MdOutlineMailOutline, MdFacebook } from "react-icons/md";
+import { HiOutlinePhone } from "react-icons/hi";
+
 export default function Footer() {
+
+  const t = useTranslations('Footer');
+
   return (
-    <div> Footer yay</div>
+    <footer className={styles.footer}>
+        <div className={styles.body}>
+            {/* nav buttons */}
+            <ul className={styles.pages}>
+                <li>
+                    <Link className={styles.link} href="/">{t('pages.page1')}</Link>
+                </li>
+                <li>
+                    <Link className={styles.link} href="/">{t('pages.page2')}</Link>
+                </li>
+                <li>
+                    <Link className={styles.link}  href="/">{t('pages.page3')}</Link>
+                </li>
+                <li>
+                    <Link className={styles.link}  href="/">{t('pages.page4')}</Link>
+                </li>
+                <li>
+                    <Link className={styles.link} href="/">{t('pages.page5')}</Link>
+                </li>
+            </ul>
+        
+            {/* contacts */}
+            <div className={styles.contact}> 
+                <h3>{t('contact.message')}</h3>
```

Comment:
I'll have you change this `<h3>` into a `<p>` and adjust the font sizing for it. It's a really tiny critique and not too big of a deal, but with headers its standard to go in order (`h1` always come before `h2` and so on, so not good practice to have a single one out of nowhere)